style.css

```diff
@@ -511,3 +519,245 @@ input:valid {
 .img-bar {
   justify-content: center;
 }
+
+@media (min-width: 768px) {
+  .container {
+    width: 100%;
+    height: 3074px;
+  }
+
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+  }
+
+  .nav img {
+    display: none;
+  }
+
+  .desk {
+    display: block;
+    margin-top: 0;
+  }
+
+  .desk ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+    width: 97%;
+    height: 72px;
+    margin-top: -10px;
+    margin-bottom: 20px;
+    align-items: center;
+    gap: 10px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .background {
+    width: 97%;
+    height: 344px;
+    background-image: none;
+  }
+
+  .background-container {
+    height: 100vh;
+    width: 100%;
+    background-image: url(./ImageP1/Header-llustration-desktop-1.png);
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    margin-top: -75px;
+    margin-left: 27px;
+    margin-bottom: 122px;
+  }
+
+  .myheader {
+    width: 70%;
+    height: 114px;
+    margin: 0 auto;
+  }
+
+  .break {
+    display: none;
+  }
+
+  p {
+    width: 70%;
+    height: fit-content;
+    font-size: 18px;
+    margin: 24px auto;
+    margin-top: 0;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  .flex-icons ul {
+    width: 200px;
+    height: 40px;
+    margin: 0 auto;
+    margin-bottom: 45px;
+    padding-left: 0;
+  }
+
+  .flex-icons li {
+    padding: 0 auto;
+  }
+
+  .chevron {
+    margin: 0 auto;
+    margin-top: 0;
+    margin-bottom: 495px;
+  }
+
+  .works {
+    margin-top: 0;
+  }
+
+  header .work {
+    width: 447px;
+    height: 48px;
+    margin-left: 497px;
+    margin-right: 496px;
+    margin-bottom: 52px;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 0;
+    margin-left: 135px;
+    padding-right: 0;
+    width: 84%;
+    height: 966px;
+    margin-bottom: 160px;
+  }
+
+  .card {
+    flex: 1;
+    margin-left: 0;
+    max-width: 327px;
+  }
+
+  .card-img {
+    max-width: auto;
+  }
+
+  .resume h2 {
+    width: 447px;
+    height: 48px;
+    margin: 0 auto;
+  }
+
+  .resume p {
+    width: fit-content;
+  }
+
+  .side-background {
+    background-image: url(./ImageP1/desk-right-background-resized.png);
+    background-repeat: no-repeat;
+    background-position: 100% 20%;
+  }
+
+  .resume {
+    background-image: url(./ImageP1/bottom-left-desk-background-resized.png);
+    background-position-y: 50%;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: 12px;
+    height: 355px;
+    width: 78%;
+    padding: 0;
+    margin-left: 136px;
+  }
```

Comment:
- [ ] Kindly position the `Languages`, `Framework` and `Skills` sections at the center of the page. Currently, they are aligned to the left part of the page. Also, reduce the gap between the cards so that your design can look identical to that of the Figma template. 👍 😎 

### Your Design

![Screenshot_2](https://user-images.githubusercontent.com/77106962/228952279-be98bc20-5783-4453-86eb-e89d0802cb20.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228952467-fc3b30cd-4f0f-43cb-aac5-95fbdb4b1de3.png)